### ISSUE TITLE
Disable "null" toast

### ISSUE THREAD

**Issue by aero31aero** [mozilla-lockwise/lockwise-android#1219]
## Steps to reproduce

- Set Lockwise as Autofill Provider on Android 11
- Tap on a textbox that doesn't need Autofill (I'm not sure on this part, happens on a lot of textboxes).

### Expected behavior

Nothing happens

### Actual behavior

A toast appears with text "null".

### Device & build information
* Device: Pixel 2 XL (taimen)
* Build version: 4.0.1
### Notes

Another person could reproduce it at https://www.reddit.com/r/android_beta/comments/hs0d6k/null_toast/fyqnoox

Attachments:

**Comment by GlenMerlin** [comment]
Can reproduce

Pixel 4 Android 11 stable
build version 4.0.1

only happens when lockwise is the currently selected autofill app

best apps I've found to test with consistent results are 
Signal Private Messenger
Firefox Android
Instagram (Direct Messaging)

Note: the toast itself is coming from android system but due to it only happening when lockwise is the default I'm led to believe something is erroring with android 11

**Comment by caeci11ius** [comment]
Confirmed occurs with lockwise as autofill provider, does not occur when provider changed to google. Problem comes back when lockwise selected again.

**Comment by AMonreale** [comment]
Same thing here, Pixel 3a with Android 11. Authy seems the more consistent app showing this toast.

**Comment by nsghub** [comment]
Same here on Pixel 3a and Android 11. Pretty anoying.

**Comment by rickhall** [comment]
Same issue for me on my Pixel 4XL...I'll have to check on my Pixel 4a later. For me, I can easily reproduce it by going into the Messages app and entering any conversation. Everything time I enter any conversation, I get a "null" popup message.

**Comment by teopb** [comment]
Happens on pixel 4a updated to android 11 today.

**Comment by marcomg** [comment]
Some issue with a pixel 3a with Android 11!

**Comment by oniGino** [comment]
Confirmed on pixel 2 w/android 11

**Comment by blitz** [comment]
Same here with a Pixel 3a and Android 11.

@eliserichards Do you someone who can take a look?

**Comment by LRomuloux** [comment]
same issue. Pixel 2 Android 11.

**Comment by JoeCactus** [comment]
Replicated on Pixel 3a XL, 4.02 not yet available on Play Store for me.

**Comment by blitz** [comment]
@Baron-Severin @eliserichards Thank you so much for fixing this! :rocket:

**Comment by rahan-crao** [comment]
Is it possible to publish a version with this fix on the Play Store ? It's been more than two weeks the fix is committed...

**Comment by oniGino** [comment]
Mozilla has pretty much abandoned this app and notes and others, who knows when they will get to pushing this

**Comment by rahan-crao** [comment]
I just made an APK because I can't wait anymore:
https://drive.google.com/file/d/1pu96SQ8QkqRqMNU1p_ERdraiB-lZrXNT/view?usp=sharing

Disclaimer:
- use it at your own risk
- you must uninstall Lockwise before installing the APK
- you will not receive Lockwise update anymore from the Play Store since it is an external APK

**Comment by GlenMerlin** [comment]
This is taking mozilla so long to just push it too the playstore I decided to switch over to bitwarden its a shame

**Comment by constructor-s** [comment]
> I just made an APK because I can't wait anymore:
> https://drive.google.com/file/d/1pu96SQ8QkqRqMNU1p_ERdraiB-lZrXNT/view?usp=sharing
> 
> Disclaimer:
> 
> * use it at your own risk
> * you must uninstall Lockwise before installing the APK
> * you will not receive Lockwise update anymore from the Play Store since it is an external APK

Sorry if I missed it but @rahan-crao how did you build it? Just clone in Android Studio and build the release version? No additional modification/external files/libraries required?

**Comment by rahan-crao** [comment]
> Sorry if I missed the build instructions but @rahan-crao how did you build it? Just clone in Android Studio and build the release version? No additional modification/external files/libraries required?

Android SDK 28 must be installed on your computer. I launched `gradlew.bat` which downloaded all dependencies. After the I used Android Studio to build the APK. Don't update the gradle plugin with Android Studio, otherwise you will not be able to build Lockwise any more.

**Comment by caeci11ius** [comment]
Just a suggestion while everyone is waiting for this to roll out.

Just disable lockwise autofill. The Firefox internal autofill still works fine for websites, so you'll only lose autofill in apps. You can manually copy password from lockwise for these. Sure, if you're regularly logging in and out of apps this will be annoying, but if you're like me, most of your logins will be to websites and unaffected. At least you'll be rid of the annoying toast popup in the meantime, and it's easy to turn it back on when the update finally rolls out.

**Comment by Cubox-** [comment]
It has been a month since the issue was fixed in this repo. There is no APK available through the releases section of this repo, nor any updates through the Play Store.

@eliserichards Is there any reason why this delay? All the Android 11 users of Lockwise would love to get the updated code. Thanks in advance

**Comment by Cubox-** [comment]
I have sent an email to the lockwise support email.

We’ve received your support request and opened a ticket (11818). We are processing your request.

I hope that'll help get someone on the team to just build a release

**Comment by rickhall** [comment]
The sad part is, LockWise is one of the more useful and important things Mozilla has released in a while. If it dies, I will be pretty bummed. Apparently, the writing is on the wall.

**Comment by daphnei-mee** [comment]
The app was last updated June 29th 2020 according to the playstore. Is it completely abandoned?

**Comment by Cubox-** [comment]
I have sent a tweet to the Mozilla twitter account, and send messages through Matrix (chat.mozilla.org) on the general channel and the lockwise channel, to raise awareness of the need for a new release.

**Comment by Webbeh** [comment]
I compiled a version of it with the null toast problem fixed. 

Since I had to self sign this Apk, it is blocked by play protect (they don't know my certificate). You will have to do a full install (remove the previous version) for it to install. 

https://geeq.ch/index.php/s/Kp5ffjTSW7oXMic

**Comment by Cubox-** [comment]
When the application signature changes key, the application won't update in place and requires a reinstall. That's normal and not due to Play Protect

Also, when you generate those APKs, make sure they are the release mode, and not the debug one. That requires the creation of a keystore, which is not needed for debug variants.
If on login you see something like "USE TEST DATA" in red at the bottom of the app, that's debug mode.

**Comment by Webbeh** [comment]
#response_container_BBPPID{font-family: initial; font-size:initial; color: initial;}  I know don't worry, I've been using the app for a few weeks, just uploaded the wrong version of the apk.                                                                                                                                                                                                                                                                                                                         From: notifications@github.comSent: 15 November 2020 07:34To: lockwise-android@noreply.github.comReply to: reply@reply.github.comCc: info@geeq.ch; comment@noreply.github.comSubject: Re: [mozilla-lockwise/lockwise-android] Disable "null" toast (#1219)  
When the application signature changes key, the application won't update in place and requires a reinstall. That's normal and not due to Play Protect
Also, when you generate those APKs, make sure they are the release mode, and not the debug one. That requires the creation of a keystore, which is not needed for debug variants.
If on login you see something like "USE TEST DATA" in red at the bottom of the app, that's debug mode.

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by Webbeh** [comment]
https://geeq.ch/index.php/s/FbRft7AEydgLAMc

**Comment by st3fan** [comment]
I just filed the following issue to get 4.0.2 out of the door. https://github.com/mozilla-lockwise/lockwise-android/issues/1239
